interfaces/utils/utils_test.go

```diff
@@ -23,56 +23,12 @@ import (
 	"testing"
 
 	. "gopkg.in/check.v1"
-
-	"github.com/snapcore/snapd/testutil"
 )
 
 func Test(t *testing.T) {
 	TestingT(t)
 }
 
-type utilsSuite struct {
-	testutil.BaseTest
-}
+type utilsSuite struct{}
```

Comment:
There don't seem to be many tests left in the suite right now.

Reply:
That's true :-) I kept it because there is still one function here and it feels like someone should write some tests for it.